projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -43,15 +52,22 @@ public ProjectCreationService(ProjectRepository projectRepository) {
    */
   public Result<Project, ApplicationException> createProject(String code, String title, String objective,
       String experimentalDesign, String sourceOffer, PersonReference projectManager,
-      PersonReference principalInvestigator, PersonReference responsiblePerson) {
+      PersonReference principalInvestigator, PersonReference responsiblePerson,
+      List<Species> speciesList, List<Analyte> analyteList, List<Specimen> specimenList) {
+
     try {
-      Project project;
-      project = createProject(code, title, objective, experimentalDesign,
+      Project project = createProject(code, title, objective, experimentalDesign,
           projectManager, principalInvestigator, responsiblePerson);
+      projectRepository.add(project);
       Optional.ofNullable(sourceOffer)
           .flatMap(it -> it.isBlank() ? Optional.empty() : Optional.of(it))
           .ifPresent(offerIdentifier -> project.linkOffer(OfferIdentifier.of(offerIdentifier)));
-      projectRepository.add(project);
+
+      Experiment experiment = Experiment.createForProject(project, analyteList, specimenList,
+          speciesList);
+      experimentRepository.add(experiment);
+      project.linkExperiment(experiment.experimentId());
```

Comment:
1.) `Linking` is not domain vocabulary, i suggest a simple `addExperiment`. 
2.) What is you strategy if the addition fails? Then you have stored a false assumption an experiment is part of a project, but it is not bilateral. I suggest to wrap this in a method and try catch with a revert strategy.